skimage/registration/_phase_cross_correlation.py

```diff
@@ -117,9 +117,16 @@ def _compute_error(cross_correlation_max, src_amp, target_amp):
     target_amp : float
         The normalized average image intensity of the target image
     """
-    error = 1.0 - cross_correlation_max * cross_correlation_max.conj() / (
-        src_amp * target_amp
-    )
+    amp = src_amp * target_amp
+    if amp == 0:
+        warnings.warn(
+            "Could not determine RMS error for normalized average image intensity "
+            f"of {amp}, either the reference or moving image may be empty",
```

Comment:
```suggestion
            f"of {amp}. Either the source or target image may be empty.",
```
Keeping the terms used in the docstring? Unless we want to take a step in [this](https://github.com/scikit-image/scikit-image/pull/7255#issue-2017120066) direction, maybe using "reference/target" then...

Reply:
Here `amp` is 0, so is it worth using the `{amp}` variable to display it? If the product is zero, (at least) one of the normalized average image intensities is zero, but technically it's either `src_amp` or `target_amp`...

"Could not determine RMS error between images of normalized average image intensities `{src_amp}` and `{target_amp}`..."

or, simply,

"Could not compute RMS error. Either the reference or target image may be empty."

?

Reply:
The terms come from the docstring of `phase_cross_correlation` rather than the internal function `_compute_error`. The former is also were `stacklevel=4` is pointing at. You are right though, no need to use `{amp}` with the current implementation.